src/oss/confluent.sh

```diff
@@ -1053,6 +1076,11 @@ connect_restart_command() {
     echo "Not implemented yet!"
 }
 
+dynamic_cli_command() {
+    local command ="${1}"
```

Comment:
haa..thanks for catching it...this was accidental ..it wasn't there in the original test script I had :)